membership-attribute-service/app/services/AttributesMaker.scala

```diff
@@ -79,6 +80,14 @@ class AttributesMaker extends LoggingWithLogstashFields{
       }
     }
 
+    def paidSubs = subs.filter{
```

Comment:
I think this can all be removed?

Reply:
ahh i thought i'd unticked this from the commit - removed now - thanks